ansible/files/envoy_config/lds.supabase.yaml

```diff
@@ -82,6 +82,27 @@ resources:
                                         name: ':path'
                                         string_match:
                                           contains: apikey=supabase_admin_key
+                        origin_protection_key_missing:
+                          permissions:
+                            - any: true
+                          principals:
+                            - not_id:
+                                or_ids:
+                                  ids:
+                                    - header:
+                                        name: sb-opk
+                                        present_match: true
+                        origin_protection_key_not_valid:
+                          permissions:
+                            - any: true
+                          principals:
+                            - not_id:
+                                or_ids:
+                                  ids:
+                                    - header:
+                                        name: sb-opk
+                                        string_match:
+                                          exact: supabase_origin_protection_key
```

Comment:
Replaced by the init scripts here: https://github.com/supabase/infrastructure/pull/20665/files#diff-a5d511773da8bfc9e40cc61e37c5d1c7f0eb22f7621975238124eef85bc76af6R75